From c4d0f332457e3b01288f06ba484cab23e8e77cc2 Mon Sep 17 00:00:00 2001
From: Quentin Casasnovas <quentinx.casasnovas@intel.com>
Date: Wed, 21 Aug 2013 15:33:57 +0200
Subject: [PORT FROM MAIN]wifi: Add support for LNP

BZ: 125764 143894

Change-Id: I9e933a6d4a365aa0cf9fbfa37152ac90c905e8f0
Orig-Change-Id: I65eb7084f26ab461bc308219c206ef295b62f8ba
Category: device enablement
Domain: CWS.WIFI-Intel_LNP
Origin: internal
Upstream-Candidate: no,proprietary
Signed-off-by: Quentin Casasnovas <quentinx.casasnovas@intel.com>
---
 wifi/Android.mk          |    4 ++++
 wifi/vendors/vendors.def |    4 ++++
 2 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/wifi/Android.mk b/wifi/Android.mk
index ef6ffc7..5e50a49 100644
--- a/wifi/Android.mk
+++ b/wifi/Android.mk
@@ -58,6 +58,10 @@ ifneq (,$(filter wifi_ti%,$(PRODUCTS.$(INTERNAL_PRODUCT).PRODUCT_PACKAGES)))
 	LOCAL_CFLAGS += -DWIFI_GLUE_WITH_TI
 endif
 
+ifneq (,$(filter wifi_intel%,$(PRODUCTS.$(INTERNAL_PRODUCT).PRODUCT_PACKAGES)))
+	LOCAL_CFLAGS += -DWIFI_GLUE_WITH_INTEL
+endif
+
 ifneq (,$(filter wifi_mtk%,$(PRODUCTS.$(INTERNAL_PRODUCT).PRODUCT_PACKAGES)))
 
 	LOCAL_CFLAGS += -DWIFI_GLUE_WITH_MTK
diff --git a/wifi/vendors/vendors.def b/wifi/vendors/vendors.def
index 521c5e2..b602abb 100644
--- a/wifi/vendors/vendors.def
+++ b/wifi/vendors/vendors.def
@@ -26,6 +26,10 @@ WIFI_GLUE(BCM, bcm_load_driver, bcm_unload_driver, bcm_switch_driver_mode,
 WIFI_GLUE(TI, NULL, NULL, NULL, NULL, NULL, NULL)
 #endif
 
+#ifdef WIFI_GLUE_WITH_INTEL
+WIFI_GLUE(INTEL, NULL, NULL, NULL, NULL, NULL, NULL)
+#endif
+
 #ifdef WIFI_GLUE_WITH_BC
 /*
  * Firmware switching.  These are called from
-- 
1.7.4.1

